### ISSUE TITLE
Start From Template should use correct useCase

### ISSUE THREAD

**Issue by quasarchimaere** [researchstudio-sat/webofneeds#2715]
We need to add a specific set of types within the content of each usecase so we can extract the proper usecase to use within create-post.js (basically we need to find the useCase by "fromNeed")

~~If we can't find the proper useCase then we should extract all the set details (seeks and content) from the "fromNeed" and display their respective pickers instead -> this should not be happen alot though, be aware that the details to choose From might hide within a specific useCase (not all the available details are within the customUseCase)~~
will be done within a different issue, lower priority since it should not happen after this is fixed anyway
blocked by #2714
